protobuf-bean/src/main/java/io/apicurio/registry/examples/simple/protobuf/ProtobufBeanExample.java

```diff
@@ -196,4 +197,25 @@ public static final void main(String [] args) throws Exception {
         return consumer;
     }
 
+    public static void configureSecurityIfPresent(Properties props) {
+        final String tokenEndpoint = System.getenv(SerdeConfig.AUTH_TOKEN_ENDPOINT);
+        if (tokenEndpoint != null) {
+
+            final String authClient = System.getenv(SerdeConfig.AUTH_CLIENT_ID);
+            final String authSecret = System.getenv(SerdeConfig.AUTH_CLIENT_SECRET);
+
+            props.putIfAbsent(SerdeConfig.AUTH_CLIENT_SECRET, authSecret);
+            props.putIfAbsent(SerdeConfig.AUTH_CLIENT_ID, authClient);
+            props.putIfAbsent(SerdeConfig.AUTH_TOKEN_ENDPOINT, tokenEndpoint);
+            props.putIfAbsent(SaslConfigs.SASL_MECHANISM, "OAUTHBEARER");
+            props.putIfAbsent(SaslConfigs.SASL_LOGIN_CALLBACK_HANDLER_CLASS, "io.strimzi.kafka.oauth.client.JaasClientOauthLoginCallbackHandler");
+            props.putIfAbsent("security.protocol", "SASL_SSL");
+
+            props.putIfAbsent(SaslConfigs.SASL_JAAS_CONFIG, String.format("org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule required " +
+                    "  oauth.client.id=%s " +
+                    "  oauth.client.secret=%s " +
+                    "  oauth.token.endpoint.uri=\"%s\" ;", authClient, authSecret, tokenEndpoint));
+        }
+    }
```

Comment:
Is this one not being called yet?

Reply:
I just forgot to update this one with the method call. Fixed.